app/views/incomeSources/manage/ManageIncomeSourceDetails.scala.html

```diff
@@ -172,8 +172,8 @@
             <dd class="govuk-summary-list__value" id="business-accounting-method">@{
                 viewModel.businessAccountingMethod match {
                     case Some(x) => x match {
-                        case value if value == "cash" => getMessage("cash-accounting")
-                        case value if value == "accruals" => getMessage("traditional-accounting")
+                        case value if !value => getMessage("cash-accounting")
```

Comment:
you can move this code to the level of viewModel, ie. create method which will return key
so you call call like:
viewModel.businessAccountingMethodAsKey(  getMessage(_) )